### PR TITLE
fixes after CompanytypeClient cleanup and depracted codes

### PR DESCRIPTION
**Pull request by gengjozsef**

### CODE REVIEW

File: src/FondOfSpryker/Glue/CompanyUserCompanyTypeRolesRestApi/Dependency/Client/CompanyUserCompanyTypeRolesRestApiToCompanyTypeClientBridge.php [hunk 1]
```diff
@@ -31,4 +32,15 @@ public function getCompanyTypeById(
     ): CompanyTypeTransfer {
         return $this->companyTypeClient->getCompanyTypeById($companyTypeTransfer);
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\CompanyTypeTransfer $companyTypeTransfer
+     *
+     * @return \Generated\Shared\Transfer\CompanyTypeTransfer
```
Comment (daniel-rose) [inline src/FondOfSpryker/Glue/CompanyUserCompanyTypeRolesRestApi/Dependency/Client/CompanyUserCompanyTypeRolesRestApiToCompanyTypeClientBridge.php]: wrong return type in phpdoc

File: src/FondOfSpryker/Glue/CompanyUserCompanyTypeRolesRestApi/Processor/Validator/CompanyUserCompanyTypeRolesSearchValidator.php [hunk 2]
```diff
@@ -85,8 +87,11 @@ public function validate(CompanyUserTransfer $companyUserTransfer): bool
             return false;
         }
 
-        if ($this->hasCompanyUserValidCompanyTypeRoles($companyRoleCollectionTransfer, $companyTypeTransfer) === false) {
-            return false;
+        if ($this->hasCompanyUserValidCompanyTypeRoles(
```
Comment (daniel-rose) [inline src/FondOfSpryker/Glue/CompanyUserCompanyTypeRolesRestApi/Processor/Validator/CompanyUserCompanyTypeRolesSearchValidator.php]: return $this->hasCompanyUserValidCompanyTypeRoles($companyRoleCollectionTransfer, $companyTypeResponseTransfer->getCompanyTypeTransfer());

**Review by daniel-rose — CHANGES_REQUESTED** [review]

File: src/FondOfSpryker/Glue/CompanyUserCompanyTypeRolesRestApi/Dependency/Client/CompanyUserCompanyTypeRolesRestApiToCompanyTypeClientBridge.php (hunk 1 above)
Comment (gengjozsef) [reply src/FondOfSpryker/Glue/CompanyUserCompanyTypeRolesRestApi/Dependency/Client/CompanyUserCompanyTypeRolesRestApiToCompanyTypeClientBridge.php]: done

File: src/FondOfSpryker/Glue/CompanyUserCompanyTypeRolesRestApi/Processor/Validator/CompanyUserCompanyTypeRolesSearchValidator.php (hunk 2 above)
Comment (gengjozsef) [reply src/FondOfSpryker/Glue/CompanyUserCompanyTypeRolesRestApi/Processor/Validator/CompanyUserCompanyTypeRolesSearchValidator.php]: done

**Review by daniel-rose — APPROVED** [review]

**Pull request merged**